tests-functional/setup.cfg

```diff
@@ -0,0 +1,5 @@
+[pytest]
+addopts=-n=auto --verbose -r=a --driver=Firefox --junit-xml=results/junit.xml --html=results/index.html
+base_url=https://www.mozilla.org/en-US/firefox/47.1/firstrun/
```

Comment:
Typically `base_url` would be `https://www.mozilla.org` and the remainder would be in the `URL_TEMPLATE` for the page object, allowing for the test to specify things like locale, and (in this case) version. See http://pypom.readthedocs.io/en/latest/user_guide.html#url-templates for details.